gsy_e_sdk/setups/asset_api_scripts/redis_basic_strategies.py

```diff
@@ -0,0 +1,198 @@
+                                    )
+                        load_strategy.append(buy_rate)
+                    else:
+                        buy_rate = (market_maker_rate +
+                                    self.asset_strategy[area_uuid]["fee_to_market_maker"])
+                        load_strategy.append(buy_rate)
+                self.asset_strategy[area_uuid]["buy_rates"] = load_strategy
+
+            # Generation strategy
+            if "available_energy_kWh" in area_dict["asset_info"]:
+                gen_strategy = []
+                for tick in range(0, TICK_DISPATCH_FREQUENCY_PERCENT):
+                    if tick < TICK_DISPATCH_FREQUENCY_PERCENT - 2:
+                        sell_rate = (market_maker_rate +
+                                     self.asset_strategy[area_uuid]["fee_to_market_maker"] -
+                                     (market_maker_rate +
+                                      2 * self.asset_strategy[area_uuid]["fee_to_market_maker"] -
+                                      fit_rate) * (tick / TICK_DISPATCH_FREQUENCY_PERCENT)
+                                     )
+                        gen_strategy.append(max(0, sell_rate))
+                    else:
+                        sell_rate = fit_rate - (
+                            self.asset_strategy[area_uuid]["fee_to_market_maker"])
+                        gen_strategy.append(max(0, sell_rate))
+                self.asset_strategy[area_uuid]["sell_rates"] = gen_strategy
+
+            # Storage strategy
+            if "used_storage" in area_dict["asset_info"]:
+                batt_buy_strategy = []
+                batt_sell_strategy = []
+                for tick in range(0, TICK_DISPATCH_FREQUENCY_PERCENT):
+                    buy_rate = (fit_rate -
+                                self.asset_strategy[area_uuid]["fee_to_market_maker"] +
+                                (med_price -
+                                 (fit_rate -
+                                  self.asset_strategy[area_uuid]["fee_to_market_maker"]
+                                  )
+                                 ) * (tick / TICK_DISPATCH_FREQUENCY_PERCENT)
+                                )
+                    batt_buy_strategy.append(buy_rate)
+                    sell_rate = (market_maker_rate +
+                                 self.asset_strategy[area_uuid]["fee_to_market_maker"] -
+                                 (market_maker_rate +
+                                  self.asset_strategy[area_uuid]["fee_to_market_maker"] -
+                                  med_price) * (tick / TICK_DISPATCH_FREQUENCY_PERCENT)
+                                 )
+                    batt_sell_strategy.append(sell_rate)
+                self.asset_strategy[area_uuid]["buy_rates"] = batt_buy_strategy
+                self.asset_strategy[area_uuid]["sell_rates"] = batt_sell_strategy
+
+    def post_bid_offer(self, rate_index=0):
```

Comment:
Nit: we've decided to rename "bid/offer" into one name ("order"). So this might become shorter:
```suggestion
    def post_order(self, rate_index=0):
```